src/dso_api/settings.py

```diff
@@ -1,21 +1,41 @@
 import logging
 import os
 import sys
+from enum import Enum
+from pathlib import Path
+from typing import Optional
 
 import environ
 import sentry_sdk
 import sentry_sdk.utils
 from corsheaders.defaults import default_headers
+from django.core.exceptions import ImproperlyConfigured
+from opencensus.trace import config_integration
 from sentry_sdk.integrations.django import DjangoIntegration
 from sentry_sdk.integrations.logging import LoggingIntegration
 
+
+class CloudEnv(Enum):
+    CLOUDVPS_DOCKER = "cloudvps_docker"
+    AZURE_ACI = "azure_aci"
+    AZURE_AKS = "azure_aks"
+
+    def is_azure(self):
+        return self in (CloudEnv.AZURE_ACI, CloudEnv.AZURE_AKS)
+
```

Comment:
Erg cool, tegelijk een beetje information leakage en maakt het minder herbruikbaar voor anderen. Misschien gewoon een string laten?

Reply:
This has the advantage that the strings are verified by virtue of https://github.com/Amsterdam/dso-api/pull/224/files#diff-f1b9d231aef12e16d24d3da417f90c4471fd4d9a213446ed5579bef76d50a992R34

Meaning if the user provides `CLOUD_ENV=fubar` the call `CloudEnv(env.str("CLOUD_ENV", "cloudvps_docker"))` will fail.

Reply:
BTW not sure what you mean with information leakage. What are we leaking? Possible values? But we leak that anyway when we later on test for specific values, regardless of whether they are string or enums.